<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>641</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>295</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="horizontalSpacing">
     <number>1</number>
    </property>
    <property name="verticalSpacing">
     <number>3</number>
    </property>
    <item row="0" column="1">
     <widget class="pboman3::TreeWidget" name="treeWidget">
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::IgnoreAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="animated">
       <bool>false</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>428</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSelectionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSelectionRename"/>
    <addaction name="separator"/>
    <addaction name="actionSelectionCut"/>
    <addaction name="actionSelectionCopy"/>
    <addaction name="actionSelectionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelectionDelete"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileOpen">
   <property name="text">
    <string>Open..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save as..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionFileExit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSelectionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Return, Enter</string>
   </property>
  </action>
  <action name="actionSelectionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSelectionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSelectionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelectionRename">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionSelectionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pboman3::TreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>treewidget.h</header>
   <slots>
    <signal>dragStarted(QList&lt;PboPath&gt;)</signal>
    <signal>dragDropped(PboPath,QMimeData*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionFileOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFileOpenClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFileSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFileSaveClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelectionDelete</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSelectionDeleteClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelectionCopy</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSelectionCopyClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>MainWindow</receiver>
   <slot>treeContextMenuRequested(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>dragStarted(QList&lt;PboPath&gt;)</signal>
   <receiver>MainWindow</receiver>
   <slot>treeDragStartRequested(QList&lt;PboPath&gt;)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>dragDropped(const PboPath, const QMimeData*)</signal>
   <receiver>MainWindow</receiver>
   <slot>treeDragDropped(const PboPath, const QMimeData*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>treeSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelectionPaste</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSelectionPasteClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelectionCut</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSelectionCutClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onFileOpenClick()</slot>
  <slot>onFileSaveClick()</slot>
  <slot>onSelectionDeleteClick()</slot>
  <slot>onSelectionCopyClick()</slot>
  <slot>treeContextMenuRequested(QPoint)</slot>
  <slot>treeDragStartRequested(QList&lt;PboPath&gt;)</slot>
  <slot>treeDragDropped(PboPath,QMimeData*)</slot>
  <slot>treeSelectionChanged()</slot>
  <slot>onSelectionPasteClick()</slot>
  <slot>onSelectionCutClick()</slot>
 </slots>
</ui>
